app/handler/tag.go

```diff
@@ -0,0 +1,58 @@
+package handler
+
+import (
+	"strconv"
+
+	"github.com/dinever/golf"
+	"github.com/dingoblog/dingo/app/model"
+)
+
+func registerTagHandlers(app *golf.Application, routes map[string]map[string]interface{}) {
+	app.Get("/api/tags", APITagsHandler)
+	routes["GET"]["tags_url"] = "/api/tags"
+
+	app.Get("/api/tags/:tag_id", APITagHandler)
+	routes["GET"]["tag_url"] = "/api/tags/:tag_id"
+
+	app.Get("/api/tags/slug/:slug", APITagSlugHandler)
+	routes["GET"]["tag_slug_url"] = "/api/tags/:slug"
```

Comment:
Did you mean to document the route here as `/api/tags/slug/:slug` so it matches the route above?